invenio/legacy/dbquery_mysql.py

```diff
@@ -535,3 +536,28 @@ def real_escape_string(unescaped_string, run_on_slave=False):
     escaped_string = connection_object.escape_string(unescaped_string)
     return escaped_string
 
+
+def aes_encrypt(field, password):
```

Comment:
The `aes_encrypt()` is only useful to make compatible the legacy code with postgresql until we'll finish to port all legacy to v2.0.
So, @lnielsen code is welcome! XD